mmv1/products/iam3/ProjectsPolicyBinding.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'ProjectsPolicyBinding'
+description: A policy binding to a Project
+references:
+  guides:
+    'Apply a policy binding': 'https://cloud.google.com/iam/docs/principal-access-boundary-policies-create#create_binding'
+  api: 'https://cloud.google.com/iam/docs/reference/rest/v3beta/projects.locations.policyBindings'
+min_version: 'beta'
+id_format: 'projects/{{project}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+base_url: 'projects/{{project}}/locations/{{location}}/policyBindings'
+self_link: 'projects/{{project}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+create_url: 'projects/{{project}}/locations/{{location}}/policyBindings?policyBindingId={{policy_binding_id}}'
+update_verb: 'PATCH'
+update_mask: true
+import_format:
+  - 'projects/{{project}}/locations/{{location}}/policyBindings/{{policy_binding_id}}'
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+custom_code:
+  post_delete: 'templates/terraform/post_delete/sleep.go.tmpl'
+autogen_async: true
+async:
+  actions: ['create', 'delete', 'update']
+  type: 'OpAsync'
+  operation:
+    base_url: '{{op_id}}'
+    path: 'name'
+    wait_ms: 1000
+  result:
+    path: 'response'
+    resource_inside_response: true
+  error:
+    path: 'error'
+    message: 'message'
+examples:
+  - name: 'iam_projects_policy_binding'
+    min_version: 'beta'
+    primary_resource_id: 'my-project-binding'
+    test_env_vars:
+      org_id: 'ORG_ID'
+    vars:
+      pab_policy_id: 'my-pab-policy'
+      display_name: 'test project binding'
+      project_binding_id: 'test-project-binding'
+parameters:
+  - name: 'location'
+    type: String
+    description: |
+      The location of the Policy Binding
+    url_param_only: true
+    required: true
+    immutable: true
+  - name: 'policyBindingId'
+    type: String
+    description: |
+      The Policy Binding ID.
+    url_param_only: true
+    required: true
+    immutable: true
+properties:
+  - name: 'name'
+    type: String
+    description: |
+      The name of the policy binding in the format `{binding_parent/locations/{location}/policyBindings/{policy_binding_id}`
+    output: true
+  - name: 'uid'
+    type: String
+    description: |
+      Output only. The globally unique ID of the policy binding. Assigned when the policy binding is created.
+    output: true
+  - name: 'etag'
+    type: String
```

Comment:
We should omit this or make it the (unfortunately underdocumented...) Fingerprint type: https://github.com/GoogleCloudPlatform/magic-modules/blob/d5e86ba02cb336a9d7819510a0b265de804c8757/mmv1/api/type.go#L705

Reply:
good to know. will do